webhooks/github_event_handler.py

```diff
@@ -16,9 +16,9 @@
 
 class GithubEventHandler(object):
 
-    def __init__(self, config=None, jenkins=None):
+    def __init__(self, config=None, jenkins_host=None):
         self.__config = config
-        self.__jenkins = jenkins
+        self.__jenkins = jenkins_host
```

Comment:
`jenkins` args was used to inject the dependecy. Are you sure we want to rename it?